web/observing_changes/observing_changes_to_light_dom_children/my_element.dart

```diff
@@ -1,27 +1,39 @@
 @HtmlImport('my_element.html')
-library web.observing_changes.observing_changes_to_light_dom_children;
+library my_element;
 
 import 'dart:html';
-
+import 'package:web_components/web_components.dart' show HtmlImport;
 import 'package:polymer/polymer.dart';
 
-@CustomTag('my-element')
+@PolymerRegister('my-element')
 class MyElement extends PolymerElement {
-  @observable String message = '';
+  int _counter = 1;
+
+  @property String message = '';
+
+  MutationObserver observer;
+  MyElement.created() : super.created();
 
-  MyElement.created() : super.created() {
-    onMutation(this).then(childrenUpdated);
+  ready() {
+    // onMutation callback isn't supported in Polymer 1.0 anymore
```

Comment:
@zoechi where did you see those docs? We should make sure it gets updated (in the main polymer js docs i only see this referenced for the `dom-*` helper elements).

In general though we may just want to remove this entire example (since its not directly supported any longer as far as I can tell).